src/types/zetkin.ts

```diff
@@ -246,13 +246,22 @@ export interface ZetkinJourneyInstance {
   assigned_to: ZetkinPerson[];
   created_at: string;
   id: number;
-  next_milestone: {
-    deadline: string;
+  journey: {
+    id: number;
     title: string;
-  } | null;
+  };
+  milestones: ZetkinJourneyMilestone[] | null;
+  next_milestone: ZetkinJourneyMilestone | null;
   people: ZetkinPerson[];
+  status: string;
```

Comment:
If we know these statuses, we can make an enum and then set the type of `status` to that enum.

Reply:
This is not the final data model, so I think it's wise to keep it as string (make minimal effort) at this point. I'm still working on the API, but right now it doesn't seem like there will be a `status` field at all. Instead, a `closed` field will either be a timestamp or `null`.

But those details should wait until the API is finalized, which it will be before next week.